Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/ConceptLookup.java

```diff
@@ -81,4 +92,25 @@
 
         return new ResponseEntity<String>(returnMsg.getXML(xmlEntries), HttpStatus.OK);
     }
+
+    @RequestMapping(value = "rest/ConceptLookup/{word}/{pos}", method = RequestMethod.GET, produces = "application/rdf+xml")
+    public @ResponseBody ResponseEntity<String> getWordNetEntryInRdf(@PathVariable("word") String word,
+            @PathVariable("pos") String pos) {
+        ConceptEntry[] entries = null;
+        try {
+            entries = dictManager.getConceptListEntriesForWord(word, pos, null);
+        } catch (LuceneException ex) {
+            return new ResponseEntity<String>(ex.getMessage(), HttpStatus.BAD_REQUEST);
+        } catch (IllegalAccessException e) {
+            return new ResponseEntity<String>(e.getMessage(), HttpStatus.BAD_REQUEST);
+        }
+        Model model = ModelFactory.createDefaultModel();
+        String syntax = "RDF/XML";
+        StringWriter out = new StringWriter();
+        for (ConceptEntry entry : entries) {
+            rdfFactory.generateRDF(model, syntax, out, entry);
```

Comment:
you probably don't want to have RDF classes in the controller. You should return a string from the RDF class or a custom object (if that works better), so that the controller doesn't have to be changed if the RDF library changes.